backend/whatsapp.py

```diff
@@ -181,6 +181,22 @@ def onMessage(self, ws, message):
                         try:
                             processedData = whatsappReadBinary(decryptedMessage, True);
                             messageType = "binary";
+
+                            # sort contacts obj{jid : name}
+                            try:
+                                if processedData[1]['type'] is "contacts":
+                                    messageType = "jsonContacts";
+                                    processedData.append(self.sortedContacts(processedData)) 
+                            except:
+                                pass
+                            # sort statuses
+                            try:
+                                if processedData[2][0][0] is "status":
+                                    processedData[2] = self.sortedStatuses(processedData)
+                                    messageType = "jsonStatuses";
+                            except:
+                                pass
```

Comment:
Are these exceptions caught as a result of, say, `processedData[1]['type']` throwing IndexError or something?

Reply:
yes, some responses don’t have a 'type' key, like 'action' response.
so sometimes it throws an error but try/except will handle that.

Reply:
> Are these exceptions caught as a result of, say, `processedData[1]['type']` throwing IndexError or something?

@quad any problem ?